docs/tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.md

```diff
@@ -0,0 +1,36 @@
+# Test duplicated regitry entries
+
+This test will check that for two monitored registries with the same name value but with different capitalisation only triggers one modified event when the registry is changed.
+
+## General info
+
+|Tier | Number of tests | Time spent|
+|:--:|:--:|:--:|
+| 0 | 1 | 25s |
+
+## Test logic
+
+The test creates two registries with the same name but different capitalization,
+modifies one of them and waits for the monitor to grab that event.
+The second call to the monitor should arise TimeoutError to be successful. This test
+ensures the windows agent doesn't duplicate alerts due to windows being case insensitive.
```

Comment:
```suggestion
ensures the Windows agent doesn't duplicate alerts due to Windows being case insensitive.
```

Reply:
Documentation changed in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/2ae38fa0530f59f33378bda5978f2bbcb45d015c)